File: WssCRM/Models/Company.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WssCRM.Models
{
    public class Company
    {
        public string Name { get; set; }
        public int id { get; set; }
        public List<Stage> stages = new List<Stage>();
        public List<Manager> managers = new List<Manager>();
        public Company(string name, int id)
        {
            Name = name;
            this.id = id;
        }
        public Company()
        {

        }
    }
}

File: WssCRM/ClientApp/src/app/Models/Company.ts
import {Stage} from './Stage'
import { Manager } from './Manager';
export class Company {

  constructor(
    public name?: string,
    public managers?: Manager[],
    public stages?: Stage[],
    public daysForAnalyze ? : number,
    public id ? : number
    
  ) {
      this.managers = [];
    this.stages = [];
    this.daysForAnalyze = 21;
    }
}
